packages/native-stack/src/views/DebugContainer.native.tsx

```diff
@@ -15,22 +18,26 @@ type ContainerProps = ViewProps & {
  * for detailed explanation.
  */
 export let DebugContainer = (props: ContainerProps) => {
-  return <View {...props} collapsable={false} />;
+  return <ScreenContentWrapper {...props} />;
```

Comment:
It is crucial for formsheet presentation to work with native-stack that the children of the Screen component are wrapped inside `ScreenContentWrapper`. This, and another change few lines below assures this.